src/main/java/com/google/sps/servlets/AvailabilityServlet.java

```diff
@@ -28,10 +28,27 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.utilities.RealAvailabilityDatastore;
+import com.google.sps.utilities.MockAvailabilityDatastore;
+import com.google.sps.utilities.AvailabilityDatastoreService;
 
 @WebServlet("/availability")
 public class AvailabilityServlet extends HttpServlet {
 
+    private AvailabilityDatastoreService datastore;
+
+    public AvailabilityServlet(){}
```

Comment:
I guess a similar comment describing the use of this would be helpful like in the search servlet.